internal/helpers/secrets_test.go

```diff
@@ -1522,3 +1523,46 @@ func TestSecretDataBuilder_WithHVSAppSecrets(t *testing.T) {
 		})
 	}
 }
+
+func TestHasOwnerLabels(t *testing.T) {
+	t.Parallel()
+
+	// label setup copied to controllers.Test_secretsPredicate_Delete()
+	require.Greater(t, len(OwnerLabels), 1, "OwnerLabels global is invalid,")
+
+	hasNotLabels := maps.Clone(OwnerLabels)
+	for k := range hasNotLabels {
+		delete(hasNotLabels, k)
+		break
+	}
+
+	tests := []struct {
+		name string
+		o    ctrlclient.Object
+		want bool
+	}{
+		{
+			name: "has",
+			o: &corev1.Secret{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: OwnerLabels,
+				},
+			},
+			want: true,
+		},
+		{
+			name: "has-not",
+			o: &corev1.Secret{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: hasNotLabels,
+				},
+			},
+			want: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Do these need to be marked with t.Parallel() as well?

Reply:
Added in acaca4fd3a4de59d1b81fe0856c9e423245dda28